src/main/flight/mixer.c

```diff
@@ -150,6 +150,7 @@ static const motorMixer_t mixerTricopter[] = {
     { 1.0f,  1.0f, -0.666667f,  0.0f },     // LEFT
 };
 
+#ifndef DISABLE_UNCOMMON_MIXERS
```

Comment:
Bu then, chance of adding support to any new F1 target is rather slim, isn't it?
That way, any new targets will be unaffected on config level by this space saving override